repair/task_manager_module.hh

```diff
@@ -223,7 +224,13 @@ public:
     size_t ranges_size() const noexcept;
 
     virtual void release_resources() noexcept override;
+private:
+    future<> maybe_update_erm(tasks::task_manager::module_ptr module, tasks::task_id task_id, table_id table_id,
+        std::unordered_set<locator::tablet_id> involved_tablets) noexcept;
+    // Requires task to have been registered in task manager.
+    void start_erm_update_layer() noexcept;
```

Comment:
I don't understand the word "layer" in this context.

Reply:
I meant to indicate that this function starts some background job

Reply:
Well, I meant _fiber_ :facepalm: